### PR TITLE
Snowfakery 0.8.1

### PR DESCRIPTION
**Pull request by prescod**
Release Snowfakery 0.8.1

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ In the beginning, programmers created the databases. Now the databases were form
 
 And so [Salesforce.org](http://salesforce.org/) said “Let there be data,” and there was Snowfakery. And it was good.
 
+## Snowfakery 0.8.1
+
+Snowfakery includes support for Python RecordTypes.
```
Comment (davisagli) [inline HISTORY.md:9]: Python?

File: HISTORY.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ In the beginning, programmers created the databases. Now the databases were form
...
+
+Snowfakery can output SQL/JSON NULL using Yaml blank fields or the YAML literal 'null'
```
Comment (davisagli) [inline HISTORY.md:11]:
```suggestion
Snowfakery can output SQL/JSON NULL using YAML blank fields or the YAML literal 'null'
```

File: HISTORY.md [hunk 3]
```diff
@@ -4,6 +4,14 @@ In the beginning, programmers created the databases. Now the databases were form
...
+
+Various performance and reliability improvements.
```
Comment (davisagli) [inline HISTORY.md:13]: Is there a good reason to be vague about it? I always hate it when release notes don't actually say what changed.

**Comment by prescod** [comment]
Okay updated and I also added a change to CONTRIBUTING which I missed before.

**Pull request merged**